docs/user-guide/configuring.md

```diff
@@ -664,6 +664,7 @@ In each case, the settings in the configuration file override default settings.
 ESLint supports configuration files in several formats:
 
 * **JavaScript** - use `.eslintrc.js` and export an object containing your configuration.
+* **JavaScript (ESM)** - same as `.eslintrc.js` but used with ES module packages
```

Comment:
Should this be `.eslintrc.cjs`? I agree with @ljharb that we could clarify this. Suggestion:

```suggestion
* **JavaScript (ESM)** - use `.eslintrc.cjs` when running ESLint in JavaScript packages that specify `"type":"module"` in their `package.json`. Note that ESLint does not support ESM configuration at this time.
```

We could then get rid of the note below.

Reply:
Assuming this is a suitable fix, this change was made and the note below was removed in 647feca